From cb182388f03b9a7d6ed6f6fb9908b0ac17753f4e Mon Sep 17 00:00:00 2001
From: Laurent Bigonville <bigon@bigon.be>
Date: Sun, 31 Aug 2014 19:02:03 +0200
Subject: [PATCH] libselinux: man: Add missing manpage links to
 security_load_policy.3

Add the missing manpage link for selinux_init_load_policy(3) and
selinux_mkload_policy(3) to security_load_policy(3)

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=753803
---
 libselinux/man/man3/selinux_init_load_policy.3 | 1 +
 libselinux/man/man3/selinux_mkload_policy.3    | 1 +
 2 files changed, 2 insertions(+)
 create mode 100644 libselinux/man/man3/selinux_init_load_policy.3
 create mode 100644 libselinux/man/man3/selinux_mkload_policy.3

diff --git a/man/man3/selinux_init_load_policy.3 b/man/man3/selinux_init_load_policy.3
new file mode 100644
index 0000000..94e8117
--- /dev/null
+++ b/man/man3/selinux_init_load_policy.3
@@ -0,0 +1 @@
+.so man3/security_load_policy.3
diff --git a/man/man3/selinux_mkload_policy.3 b/man/man3/selinux_mkload_policy.3
new file mode 100644
index 0000000..94e8117
--- /dev/null
+++ b/man/man3/selinux_mkload_policy.3
@@ -0,0 +1 @@
+.so man3/security_load_policy.3
-- 
2.1.0

